server/lib/two-factor-authentication/webauthn.ts

```diff
@@ -0,0 +1,282 @@
+import crypto from 'crypto';
+
+import { AsnParser, OctetString } from '@peculiar/asn1-schema';
+import { Certificate } from '@peculiar/asn1-x509';
+import * as simplewebauthn from '@simplewebauthn/server';
+// eslint-disable-next-line import/no-unresolved, node/no-missing-import
```

Comment:
Do we need this?

Reply:
Yes, unfortunately these linter rules are not picking up the exports declaration on this dependency's package.json :/ I havent been able to fix this by changing the rules.